buffer/src/main/java/io/netty5/buffer/api/Buffer.java

```diff
@@ -302,14 +302,14 @@ default int writableBytes() {
      * or {@code length}.
      *
      * @param srcPos The byte offset into this buffer from where the copying should start; the byte at this offset in
-     *              this buffer will be copied to the {@code destPos} index in the {@code dest} array.
+     *              this buffer will be copied to the {@code destPos} index in the {@code dest} {@link ByteBuffer}.
      * @param dest The destination byte buffer.
-     * @param destPos The index into the {@code dest} array from where the copying should start.
+     * @param destPos The index into the {@code dest} {@link ByteBuffer} from where the copying should start.
      * @param length The number of bytes to copy.
      * @throws NullPointerException if the destination buffer is null.
      * @throws IndexOutOfBoundsException if the source or destination positions, or the length, are negative,
-     * or if the resulting end positions reaches beyond the end of either this buffer, or the destination array.
-     * @throws java.nio.ReadOnlyBufferException if the destination buffer is read-only.
+     * or if the resulting end positions reach beyond the end of either this buffer or the destination {@link ByteBuffer}.
```

Comment:
Checkstyle is failing here.

```
Buffer.java:311: Line is longer than 120 characters (found 122). [LineLength]
```

Reply:
I fixed it and re-run check style on it. and during that I found some typos that are fixed in following commit.